react/__tests__/attachmentHelper.test.js

```diff
@@ -0,0 +1,316 @@
+import {
+  optionPricePerItem,
+  parentPricePerUnit,
+} from '../utils/attachmentHelper'
+
+const mock = {
```

Comment:
Use `__mocks__` to mock this.

Reply:
Yep, I would keep this in a separate file